.github/workflows/devcontainer-feature-release.yml

```diff
@@ -0,0 +1,76 @@
+name: Build dev container feature image
+
+env:
+  IMAGE_NAME: ${{ github.repository }}
+  REGISTRY: ghcr.io
+
+on:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        baseImage:
+          [
+            "ubuntu:focal",
+            "ubuntu:jammy",
+            "ubuntu:bionic",
+            "debian:11",
+            "debian:10",
+            "mcr.microsoft.com/devcontainers/base:ubuntu",
+            "mcr.microsoft.com/devcontainers/base:debian",
+          ]
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Install latest devcontainer CLI"
+        run: npm install -g @devcontainers/cli
+
+      - name: "Generating tests for 'azd' against '${{ matrix.baseImage }}'"
+        run: devcontainer features test -f azd -i ${{ matrix.baseImage }} .
+        working-directory: ./ext/devcontainer
+
+  test-scenarios:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Install latest devcontainer CLI"
+        run: npm install -g @devcontainers/cli
+
+      - name: "Testing 'azd' scenarios"
+        run: devcontainer features test -f azd --skip-autogenerated .
+        working-directory: ./ext/devcontainer
+
+  test-global:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Install latest devcontainer CLI"
+        run: npm install -g @devcontainers/cli
+
+      - name: "Testing global scenarios"
+        run: devcontainer features test --global-scenarios-only .
+        working-directory: ./ext/devcontainer
+
+  deploy:
+    if: ${{ github.ref == 'refs/heads/main' }}
+    runs-on: ubuntu-latest
+    needs: [test, test-global, test-scenarios]
+    environment:
+      name: deploy-devcontainer-feature
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Publish Features"
+        uses: devcontainers/action@v1
+        with:
+          publish-features: "true"
+          base-path-to-features: "./ext/devcontainer"
+          generate-docs: "true"
```

Comment:
Looks like you want to use the `devcontainers/action@v1` action to automatically generate docs (ie README.md)? However, you are missing some steps which helps open PRs for doc updates. 

See https://github.com/devcontainers/feature-starter/blob/main/.github/workflows/release.yaml#L26-L47

Reply:
@danieljurek - Not sure if this is something we'll have to do inside on our AzDo pipelines, due to the need to push a PR?

Reply:
Since the permissions would be scoped to the `azure-dev` repo I'm OK with adding that permission to the action.

Reply:
Also, it would be worth checking if GitHub Actions/actions-bot is allowed to open PRs for this repo. Sometimes, the org disables it, then one needs to use a bot's PAT (fine grained  preferred) or a personal PAT for the process.

(As Azure is part of Microsoft Open Source, I think it would be disabled.)

Ref: https://github.com/devcontainers/features/commit/d84e46be47867e596fdd2f2efaefb63f67597a07

Reply:
Once there's confirmation we can update the workflow accordingly.